modules/puzzles/module.py

```diff
@@ -88,6 +88,21 @@ def ascii(self, numbers):
         else:
             self.respond('Verkeerde input. Ik verwacht bijvoorbeeld \'/ascii 84,101,115,116\'.')
 
+    @command
+    def alphabet(self, items):
+        """
+        Vertaalt de alphabetindices naar alphabetletters, of de alphabetletters naar alphabetindices. De inputwaarden dienen gescheiden te worden met komma's.
+        """
+        item_list = items.split(',')
+        if all([n.isalpha() for n in item_list]):
+            result = util.characters_to_character_indices(item_list)
+            self.respond(items + ': ' + result)
+        elif all([n.isdigit() and 1 <= int(n) <= 26 for n in item_list]):
+            result = util.character_indices_to_characters(item_list)
+            self.respond(items + ': ' + result)
```

Comment:
dit gaat denk ik mis omdat result een lijst is kun je dat niet toevoegen aan een string. Je kan wel string interpolatie gebruiken `f'{items}: {result}'` of str() callen